### DISCUSSION TITLE
can kotaemon plug to a NAS and specific folders ?

### DISCUSSION THREAD

**Discussion by stanthewizzard** [Cinnamon/kotaemon#497] [General]
Hello

I'd like to know if:
- kotaemon plug to a NAS (docker mount)
- can user specific folders without upload to the service ?

Thanks

**Comment by taprosoft** [comment]
@stanthewizzard it is possible but we have to add a feature for regularly sync updated files from the target folder or NAS to perform indexing. It will be noted in the roadmap.

**Reply by stanthewizzard** [reply]
Awesome !!!
